Move examples from cog-examples to cog (#3055)
replicate/cog: examples/hello-context/README.md

## hello-context

> A simple model that takes no inputs but will echo back any context provided with the prediction as the output.

Reviewer: This README says the model takes no inputs, but `run.py` requires a `text` input and the harness supplies one. Can we make the README match the actual example?

```suggestion
A simple model that echoes the `text` input and any prediction context in the output.
```

Author: Fixed — README now says it echoes the `text` input and any prediction context.
